tools/validate_configs/test_configs/rocky-linux.yaml

```diff
@@ -53,8 +53,6 @@ deployment_groups:
       spack_url: https://github.com/spack/spack
       spack_ref: v0.17.0
       spack_cache_url:
-      - mirror_name: 'gcs_cache'
-        mirror_url: gs://example-buildcache/linux-centos7
```

Comment:
Should this have been replaced by something? I see you are removing them in several places. I am missing context.

Reply:
I removed them for testing. I can add them back in, but I'm not sure how much value these lines add to the validate configs test; other tests cover a broad range of spack-install variables.

Reply:
As we discussed offline, I left them out, and removed them consistently for all of these *-ss.yaml tests.